app/src/lib/api.ts

```diff
@@ -76,10 +76,9 @@ export interface IAPICommit {
 export interface IAPIUser {
   readonly id: number
   readonly url: string
-  readonly type: 'user' | 'org'
```

Comment:
Valid values from the Github API are `User` or `Organization` but this field did not seem to be used anywhere in the project.